package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "nomad-ui",
+  "private": true,
+  "scripts": {},
+  "packageManager": "pnpm@10.14.0+sha512.ad27a79641b49c3e481a16a805baa71817a04bbe06a38d17e60e2eaee83f6a146c6a688125f5792e48dd5ba30e7da52a5cda4c3992b9ccf333f9ce223af84748",
+  "engines": {
+    "node": "20.*",
```

Comment:
Vercel builds a UI preview with each PR. This node version will need to change to ">=22.17.1" when the Web Presence team updates the Node version in Vercel. The Web Presence team told me that they will open PRs to update the Node version in website/package.json and ui/package.json to coordinate with their update of the Vercel Node version. https://hashicorp.slack.com/archives/C01KCU4HDPY/p1755215116716779?thread_ts=1755184896.362699&cid=C01KCU4HDPY